common/src/main/java/tds/common/configuration/CacheConfiguration.java

```diff
@@ -37,23 +41,25 @@ public CacheManager cacheManager() {
     }
 
     @Bean
-    @Qualifier(CacheType.SHORT_TERM)
+    public KeyGenerator keyGenerator() {
+        return new CacheKeyGenerator();
+    }
+
+    @Bean
```

Comment:
Nope. Behind the scenes Spring will look at the cachename of the cache, not the bean id.